src/OSPSuite.UI/Views/Charts/ChartDisplayView.cs

```diff
@@ -73,52 +77,40 @@ public override void InitializeResources()
 
          _chartControl.Legend.MarkerSize = UIConstants.Chart.LEGEND_MARKER_SIZE;
          _chartControl.Legend.Direction = LegendDirection.BottomToTop;
-         xyDiagram.EnableAxisXZooming = true;
-         xyDiagram.EnableAxisYZooming = true;
          xyDiagram.ZoomingOptions.UseKeyboard = false;
          xyDiagram.EnableAxisXScrolling = false;
          xyDiagram.EnableAxisYScrolling = false;
 
-         _diagramZoomRectangleService = new DiagramZoomRectangleService(_chartControl, zoomAction);
-         _diagramZoomRectangleService.InitializeZoomFor(xyDiagram);
+         new DiagramZoomRectangleService(_chartControl, zoomAction);
 
          SetFontAndSizeSettings(ChartFontAndSizeSettings.Default);
 
          _chartControl.Series.Remove(s1);
-
          _chartControl.CrosshairEnabled = DefaultBoolean.False;
-
          _chartControl.ObjectHotTracked += (o, e) => OnEvent(() => onObjectHotTracked(e));
          _chartControl.ObjectSelected += (o, e) => OnEvent(() => onObjectSelected(e));
          _chartControl.Zoom += (o, e) => OnEvent(() => onZoom(e));
          _chartControl.Scroll += (o, e) => OnEvent(() => onScroll(e));
-
          _chartControl.CustomDrawSeriesPoint += (o, e) => OnEvent(() => drawSeriesPoint(e));
-
-         AllowDrop = true;
-
          _chartControl.Click += (o, e) => OnEvent(() => chartClick(e as MouseEventArgs));
          _chartControl.DoubleClick += (o, e) => OnEvent(() => chartDoubleClick(e as MouseEventArgs));
-
          _chartControl.MouseMove += (o, e) => OnEvent(() => OnChartMouseMove(e));
          _chartControl.DragOver += (o, e) => OnEvent(() => OnDragOver(e));
          _chartControl.DragDrop += (o, e) => OnEvent(() => OnDragDrop(e));
-         _chartControl.Resize += (o, e) => OnEvent(() => OnResize(e));
+         _chartControl.SizeChanged += (o, e) => OnEvent(onSizeChanged);
```

Comment:
hope this one would be better but not.
To be honest, reacting to the view OnResize when the Resize of the control is thrown is weird anyways so that way, it's decoupled